crates/viewer/re_dataframe_ui/src/table_blueprint.rs

```diff
@@ -65,3 +67,75 @@ pub struct TableBlueprint {
     pub sort_by: Option<SortBy>,
     pub partition_links: Option<PartitionLinksSpec>,
 }
+
+/// The blueprint for a specific column.
+// TODO(ab): these should eventually be stored in `TableBlueprint`, but is currently not strictly
+// necessary since we don't need to store the column blueprint for now.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ColumnBlueprint {
+    /// The name to use for this column in the UI.
+    ///
+    /// If `None`, the column will be named using [`default_display_name_for_column`].
+    pub display_name: Option<String>,
+    pub default_visibility: bool,
+    pub variant_ui: Option<VariantName>,
+}
+
+impl Default for ColumnBlueprint {
+    fn default() -> Self {
+        Self {
+            display_name: None,
+            default_visibility: true,
+            variant_ui: None,
+        }
+    }
+}
+
+impl ColumnBlueprint {
+    /// Same as [`Self::default()`], but returns a reference to a static instance.
+    pub fn default_ref() -> &'static Self {
+        use std::sync::OnceLock;
+        static DEFAULT: OnceLock<ColumnBlueprint> = OnceLock::new();
```

Comment:
I honestly didn't realize that rust can scope statics to functions. that's neat
A `Lazy` would be more appropriate than a `OnceLock` though